src/mockserver.js

```diff
@@ -1,9 +1,12 @@
 var config = window.__karma__.config,
-	ui5config = (config && config.openui5) || {},
+	ui5config = (config && config.openui5.config) || {},
```

Comment:
If you cange this, the next line will not work anymore. `ui5config.mockserver` will never be defined.